pkgs/development/compilers/mit-scheme/default.nix

```diff
@@ -9,7 +10,7 @@ let
          else                         "";
 in
 stdenv.mkDerivation {
-  name = "mit-scheme-${version}";
+  name = if enableX11 then "mit-scheme-x11-${version}" else "mit-scheme-${version}";
```

Comment:
Could be `name = "mit-scheme${stdenv.lib.optionalString enableX11 "-x11"}-${version}";`